includes/stringfuzz.tex

```diff
@@ -5,110 +5,119 @@ \subsubsection{Implementation and Architecture}
 
 \fuzzer{} is implemented as a Python package, and comes with several
 executables to generate, transform, and analyze \smtfull{} string and regex
-instances. It can either be installed from source, or from the Python
-PIP package repository.\footnote{The link to the
-source code will be added after double-blind review.}
-\fuzzer{} is organized to be easily extended:
-all generators and transformers are independent Python modules. To demonstrate
-its modularity, we note that the implementations of transformers in \fuzzer{}
-contain \linesPerX{} lines of code on average, whereas the whole fuzzer
-contains \linesInFuzzer{} lines. Its components are also
-implemented as \unix{} ``filters'', to enable easy
+instances. Its components are implemented as \unix{} ``filters'' to enable easy
 integration with other tools (including themselves). For example, the
 outputs of generators can be piped into transformers, and transformers
 can be chained to produce a stream of tuned inputs to a
 solver. \fuzzer{} is composed of the following tools:
+
 \begin{description}
     \item[\generator{}] \hfill \\
-        This tool generates \smt{} instances. It supports several generators and
-        options that specify its output. Details can be found in
-        table~\ref{tbl:generators}.
+    This tool generates \smt{} instances. It supports several generators and
+    options that specify its output. Details can be found in
+    Table~\ref{tbl:generators}.
     \item[\transformer{}] \hfill \\
-        This tool transforms \smt{}
-        instances. It supports several transformers and options that specify
-        its output and input, which are explained in
-        table~\ref{tbl:transformers}. Note that transformers
-        \textit{Translate} and \textit{Reverse} also preserve
-        satisfiability under certain conditions~\cite{ifaz}.
+    This tool transforms \smt{}
+    instances. It supports several transformers and options that specify
+    its output and input, which are explained in
+    Table~\ref{tbl:transformers}. Note that transformers
+    \textit{Translate} and \textit{Reverse} also preserve
+    satisfiability under certain conditions~\cite{ifaz}.
     \item[\texttt{stringstats}] \hfill \\
-        This tool takes an \smt{}
-        instance as input and outputs its properties: the number of
-        variables/literals, the max/median syntactic depth of expressions, the
-        max/median literal length, etc.
+    This tool takes an \smt{}
+    instance as input and outputs its properties: the number of
+    variables/literals, the max/median syntactic depth of expressions, the
+    max/median literal length, etc.
 \end{description}
 
+We organized \fuzzer{} to be easily extended. As evidence of our success we note
+that while the whole project 
+contains \linesInFuzzer{} lines of code, it takes an average of 
+\linesPerX{} lines of code to create a transformer. \fuzzer{} can either be 
```

Comment:
Maybe change "while the whole project contains ... a transformer." to "the whole project contains ... of code, but transformers contain only \linesPerX{} lines on average."